docs/learn/bossroom/getting-started-boss-room.md

```diff
@@ -6,200 +6,275 @@ description: Learn more about installing and running the Boss Room game sample.
-:::
+You can get the project by downloading it directly or cloning locally. However, if you use git, you must install Git LFS.
 
-## Register the project with Unity Gaming Services (UGS)
+#### Direct download
 
-This project leverages several services from UGS to facilitate connectivity between players. To use these services inside your project, you must first [create an organization](https://support.unity.com/hc/en-us/articles/208592876-How-do-I-create-a-new-Organization-) inside the **Unity Dashboard**, and enable both the [Relay](https://docs.unity.com/relay/get-started.html) and [Lobby](https://docs.unity.com/lobby/game-lobby-sample.html) services.
+You can download the latest version of Boss Room from our [Releases](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/releases) page.
 
-## Open the project
+Alternatively, you can click the green **Code** button, then select **Download Zip**. This downloadd the branch that you are currently viewing on Github.
+
+:::note
+**Note for Windows users**: Using Windows' built-in extraction tool may generate an `Error 0x80010135: Path too long` error window which can invalidate the extraction process. A workaround for this is to shorten the zip file to a single character (for example, "`c.zip`") and move it to the shortest path on your computer (most often right at `C:\`) and retry. If that solution fails, another workaround is to extract the downloaded zip file using [7zip](https://www.7-zip.org/).
+:::
 
-To open the project for the first time:
+#### Installing Git LFS to clone locally
 
-1. In your Unity Hub, double-click the project.
-1. Navigate to the Editor window.
-2. Then under the **Project** tab, go to **Assets** > **Boss Room** > **Scenes** and double-click on the **Startup** scene.
-3. Click **Play**. The Boss Room menu scene loads.
+Boss Room uses Git Large Files Support (LFS) to handle all large assets required locally. See [Git LFS installation options](https://github.com/git-lfs/git-lfs/wiki/Installation) for Windows and Mac instructions. This step is only needed if cloning locally. You can also just download the project which will already include large files.
 
-## Test multiplayer
+### Registering the project with Unity Gaming Services (UGS)
 
-To see the multiplayer functionality in action, you can either run multiple instances of the game [locally on your computer](#local-multiplayer-setup) or choose to [connect through the internet](#multiplayer-over-internet).
+Boss Room leverages several services from UGS to facilitate connectivity between players. To use these services inside your project, you must [create an organization](https://support.unity.com/hc/en-us/articles/208592876-How-do-I-create-a-new-Organization-) inside the Unity Dashboard, and enable the [Relay](https://docs.unity.com/relay/get-started.html) and [Lobby](https://docs.unity.com/lobby/game-lobby-sample.html) services. Otherwise, you can still use Boss Room without UGS.
```

Comment:
_We might make this a list (I find them easier to follow)_

Boss Room leverages several services from UGS to facilitate connectivity between players. Boss Room will still work with direct IP connections (i.e. without UGS), however if you would like to use UGS services with Boss Room you must:
- [Create an organization](https://support.unity.com/hc/en-us/articles/208592876-How-do-I-create-a-new-Organization-) inside the Unity Dashboard
- Enable [Relay](https://docs.unity.com/relay/get-started.html) services.
- Enable [Lobby](https://docs.unity.com/lobby/game-lobby-sample.html) services.

Reply:
Agreed. Just don't add too much details. BR shouldn't own UGS setup steps.